lib/improver/spotdata/extrema.py

```diff
@@ -109,8 +109,7 @@ def process(self, cube):
         period_cubes = CubeList()
         for period_start, period_end in zip(starts, ends):
             extrema_constraint = datetime_constraint(period_start, period_end)
```

Comment:
`iris.FUTURE.context(cell_datetime_objects=True)` statements (not required with Iris 2) generate an Iris deprecation error message when using Iris 2. This error message is put in the unittests `warning_list` as the first warning in the list. The 'required' unittest error message subsequently gets bumped down to the second item in the `warning_list` causing subsequent checks on the `warning_list` error message text and length of the list to fail.